plugins/quay/package.json

```diff
@@ -69,11 +69,5 @@
     "plugin"
   ],
   "homepage": "https://janus-idp.io/",
-  "bugs": "https://github.com/janus-idp/backstage-plugins/issues",
-  "scalprum": {
```

Comment:
Can you please remove this from all the other plugins as well?

Reply:
Sure